htlcswitch.go

```diff
@@ -131,10 +141,9 @@ func (h *htlcSwitch) Stop() error {
 // alternative error is returned.
 func (h *htlcSwitch) SendHTLC(htlcPkt *htlcPacket) error {
 	htlcPkt.err = make(chan error, 1)
-
 	h.outgoingPayments <- htlcPkt
-
-	return <-htlcPkt.err
+	err := <-htlcPkt.err
+	return err
```

Comment:
Hmm, I don't think we gain anything with this change -- the before and after are equivalent with the latest diff adding an additional line.